examples/n2c-miniprotocols/src/main.rs

```diff
@@ -66,7 +66,7 @@ fn main() {
 
     // we connect to the unix socket of the local node. Make sure you have the right
     // path for your environment
-    let bearer = Bearer::connect_unix("/tmp/node.socket").unwrap();
+    let bearer = Bearer::connect_tcp("/tmp/node.socket").unwrap();
```

Comment:
@Quantumplation why this change?

Reply:
Because cargo test said the method didn't exist, and so was failing; I figured the method had been renamed but not updated in the docs

Reply:
oh, I think it's because that method is compiled conditionally only on OS that supports unix sockets.

Reply:
Should I revert it? Is there a way to mark the documentation/example conditionally in a similar way?

Reply:
I've added a conditional panic to make it explicit, let me know what you think.